about.html

```diff
@@ -0,0 +1,243 @@
+      </div>
+</article>
+<article class="womenKenyaLogo">
+  <h2 class="text-center seconday-dark">Kenya Women’s Summit Logo</h2>
+  <div class="line"></div>
+  <div class="logoDescripiton">
+    <p class="text-center desktopDescription">
+      The Kenya women's summit brings together high-achieving women from all over the country. 
+      It gives focus to new ways of thinking and presents aspects we think we already understand from a new and unfamiliar perspective. 
+      This summit enables women to start a dialogue; women who shape their societies and their organisations – women who make their mark.
+      It directs attention to new markets and fields of action for companies and regions, enables the joint exploration of visions and strategies, 
+      and extends the boundaries of previous knowledge. 
+    </p>
+    <p class="text-center mobileDescription">
+      The Kenya Women’s summit covers a wide range of aspects, 
+      from the vital questions of the world economy to initiatives 
+      which demonstrate exemplary actions in times of crisis and also helps spread the information all over the country.  
+    </p>
+    <div class="mt-5 d-flex justify-content-center 
+    align-items-center womenKenyaLogoDiv">
+    <img src="./images/womenlogo3.png" alt="logo">
+  </div>
+  </div>
+  <div class="container programLogo d-flex"></div>
+</article>
+<article class="pastEvents">
+  <h2 class="text-center seconday-dark">Previous Events</h2>
+  <div class="line"></div>
+  <div class="eventDescripition">
+    <p class="text-center my-3">
+      have a look at our accomplishments from previous events and the change brought by them
+    </p>
+  </div>
+  <div class="container events d-flex">
+    <div class="event event-left d-flex justify-content-center
+    align-items-center p-5">
+    <div>
+      <h4 class="text-white eventYear">2020</h4>
+      <p class="text-white">
+        Kenya Women’s Summit 
+      </p>
+    </div>
+  </div>
+  <div class="event-right event d-flex justify-content-center
+  align-items-center p-5">
+   <div>
+    <h4 class="text-white eventYear">2021</h4>
+    <p class="text-white">
+      Kenya Women’s Summit
+    </p>
+   </div>
+ </div>
+</div>
+</article>
+<article class="partners">
+  <h2 class="text-center secondary-light">partners</h2>
+  <div class="line"></div>
+  <div class="container d-flex partnerslogos">
+      <div class="partnerLogodiv">
+          <img src="./images/partner/google.svg" alt="google">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/mozilla.svg" alt="mozilla">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/microsoft.svg" alt="microsoft">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/zoom-communications.svg" alt="zoom">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/airbnb.svg" alt="airbnb">
+      </div>
+  </div>
+</article>
```

Comment:
- [ ] Kindly this section doesn't exist on the about page, it shows on the mobile version but on the desktop version, it does not exist.

Your design
![image](https://user-images.githubusercontent.com/35535130/168910175-ef266eac-568d-485e-89ec-94e8e8dd3c97.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/168910265-2fada53d-87d6-48fa-a6ab-be33b1aed75a.png)